Cartfile.resolved

```diff
@@ -1,8 +1,8 @@
-github "amplitude/Amplitude-iOS" "a3d6720626e72a678489bce83ed9dacadb505dd7"
+github "amplitude/Amplitude-iOS" "32d36977147d9f9a6330002afdc85fe31cfbcfaa"
```

Comment:
Can you revert this guy until I can test it? `carthage update rileylink_ios` will isolate the fetch